plugins/reporters/static-html/src/main/kotlin/ReportTable.kt

```diff
@@ -35,7 +35,7 @@ import org.ossreviewtoolkit.model.config.ScopeExclude
 import org.ossreviewtoolkit.model.licenses.ResolvedLicense
 import org.ossreviewtoolkit.utils.spdx.SpdxExpression
 
-internal fun Collection<ReportTable.ResolvableIssue>.containsUnresolved() = any { !it.isResolved }
+internal fun Collection<ResolvableIssue>.containsUnresolved() = any { !it.isResolved }
```

Comment:
Commit message: The class is actually not called "ReportTableModel" anymore, but "ReportTable".